server/src/map/map.model.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose');
+
+// define the schema for our map
+const mapSchema = mongoose.Schema({
```

Comment:
not sure, but ususally schema is saved separatelly from the code. but in this case it's also fine, since it's just one schema will be)